src/iocore/cache/StripeSM.cc

```diff
@@ -214,7 +214,7 @@ StripeSM::handle_dir_clear(int event, void *data)
     if (!op->ok()) {
       Warning("unable to clear cache directory '%s'", hash_text.get());
       disk->incrErrors(op);
-      fd = -1;
+      disk->fd = -1;
```

Comment:
@moonchen Would you take a look at this change? When `CacheDisk` got error, can we make the `fd = -1` or we should keep it?